sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/simple/ArrayFunctionsTests.java

```diff
@@ -107,6 +107,54 @@ public void testArrayPositionDiffTypes() {
         );
     }
 
+    @Test
```

Comment:
is there a test for a null array?
Please note that NULL and CAST(NULL AS INTEGER ARRAY) are not the same.

Reply:
No I will add the test.